### DISCUSSION TITLE
Alias vs version

### DISCUSSION THREAD

**Discussion by SoucianceEqdamRashti** [dotnet/orleans#8652] [Q&A]
Hello,

I am a little bit confused regarding alias vs version.

I understand that when you annotate interfaces with version it is to support incremental updates. 
1. Can you annotate version on your model classes/records as well?
2. If the answer to 1) is yes, do the same recommendations hold true? That is, if I add a new property to my model, that is a new version which means it should exist in a new record with that newer version?
3. With regards to Alias, if I don't use version at all, does annotating with Alias means that it will disregard any updates and just go with latest?

Would appreciated if someone could clarify these two important concepts.

**Comment by ledjon-behluli** [comment]
@SoucianceEqdamRashti there is no connection between `Alias` and `Version`.  `Alias` is used to serialize the type or interface method applied, using the alias itself instead of relaying on the type's `FullName`.

1. No
2. Answered on number 1.
3. No connection between Alias and Version.
